examples/SCAMPy/experiments/scm_pycles_pipeline/calibrate.jl

```diff
@@ -159,6 +158,7 @@ for i in 1:N_iter
     end
```

Comment:
When I run the SCAMpy pipeline, I noticed that line 155 will throw a bounderror if perform_PCA = False. Do you want to change it to something like this? (Note it will still fail with perform_PCA = False though, but with a different error.)

```
if perform_PCA
    (sim_dirs_arr, g_ens_arr, g_ens_arr_pca) = ntuple(l->getindex.(array_of_tuples,l),3)
else
    (sim_dirs_arr, g_ens_arr) = ntuple(l->getindex.(array_of_tuples,l),2)
end
```

Reply:
I think this may be fixed in ede3645